static/js/TopicPage.jsx

```diff
@@ -697,6 +698,43 @@ const TopicSideSection = ({ title, children, hasMore }) => {
   );
 }
 
+const TopicImage = ({ topicTitle }) => {
+  // const [showMore, setShowMore] = useState(false);
+  const key = topicTitle.en;
+  const hardcodedMap = {
```

Comment:
Very quick implementation, and working with the information you have, very well done!

In the proper implementation, the topic image will inherit the image will receive the 
```{'photoLink':'', 
'enCaption':'', 
'heCaption': ''}```
 as a prop. So already at this point, lets have the hardcodedMap external to the component, and receive the data as a prop